sdk/src/main/scala/com/horizen/block/SidechainBlock.scala

```diff
@@ -35,6 +36,7 @@ class SidechainBlock (
                        val sidechainTransactions: Seq[SidechainTransaction[Proposition, NoncedBox[Proposition]]],
                        val forgerBox: ForgerBox,
                        val vrfProof: VrfProof,
+                       val vrfProofHash: VrfProofHash,
```

Comment:
considering that vrfProofHash can be extracted by the vrfProof, storing it in any block would be a big waste of network resources. Moreover, considering our discussion about optimizing NonceConsensusEpochInfo calculation, the retrieval of the data to calculate the VRF Hash output from the vrfProof will be efficient enough to do it at block validation time and at block info calculation.